### ISSUE TITLE
Feature Request: Display certain entries in bold or with an asterisk

### ISSUE THREAD

**Issue by box293** [FreeRDP/Remmina#1055]
I had a suggestion for the toolbar applet. I would like to be able to have some entries appear as bold or start with an asterisk to make it easy to see them when navigating the menu. 

For example, from this screenshot you can see I often have multiple entries for the same machine ... so I can connect using different resolutions.

![tooltip_002](https://cloud.githubusercontent.com/assets/8356113/20545592/1c8c59fa-b164-11e6-8695-cad2fba21f2e.png)

Usually there is one of those entries that I use 90% of the time, I would like that entry to appear as bold, or perhaps start as an asterisk. So perhaps in that list I would see these entries marked like that:

METABOX - 1190X1850 - LAN
W10DEV - 1190X1850
WSE02 - 1190X102

It would just make it quicker to navigate. Obviously there would be some sort of "favourite" checkbox in an entry to define it as one of those.

While I've been creating this feature request I had an additional idea. Perhaps all favourite entries could also appear in an separate group called "Favouries" or something like that, so you could only see the ones that are favourites. I have 222 entries in 16 separate groups, so having an auto populated group appear in the menu would also help (but still want them to appear in their defined group).

Hope that makes sense.

**Comment by antenore** [comment]
Which version of Remmina are you using?
Since 1.2.0.rcgit.13 (even better1.2.0.rcgit.14) ,  you'll have all the most used connections on the top of the list

**Comment by box293** [comment]
I'm on 1.2.0-rcgit-15 on Linux Mint 17.3.

All my connections appear in their groups.

![menu_001](https://cloud.githubusercontent.com/assets/8356113/20552290/93cd53be-b19d-11e6-9ef3-aadc4c2ba809.png)

**Comment by antenore** [comment]
Now I've got it, sorry.

I want to discuss this with @giox069 , because we are not yet sure we want/can support the remmina applet, due to different desktop environment strategies regarding the toolbar applets.

We are thinking about alternatives. I'll be back to ASAP.

**Comment by giox069** [comment]
Yes, the so called "systray menu" is a feature which is not standardized and poorly supported across various desktop environment [(notes on our wiki)](https://github.com/FreeRDP/Remmina/wiki/Systray-menu). It has been deprecated by Gnome Shell and GTK (GtkStatusIcon). Unity 7 and KDE are currently supporting it via canonical's [libappindicator](https://launchpad.net/libappindicator). Unknown support for Unity8.

Our plan is to remove that systray menu, or better, move it inside the toolbar in the remmina connection window.

In the meanwhile:

1. as written by @antenore , in the main remmina connection window, you can sort your hosts by "last use date/time", just click on its column. Frequently used hosts will appear at the top or at the bottom. This works only on the main remmina window, it does not work on the systray menu.
2. We can also modify the current systray menu. This can be discussed here with some other posts. But... have you tried to add just an asterisk to the left of the connection name ?

**Comment by box293** [comment]
> But... have you tried to add just an asterisk to the left of the connection name ?

That is probably the easiest way, due to alphanumeric sorting I would need to put it after the name perhaps so connections are still grouped.

> Yes, the so called "systray menu" is a feature which is not standardized and poorly supported across various desktop environment (notes on our wiki). It has been deprecated by Gnome Shell and GTK (GtkStatusIcon). Unity 7 and KDE are currently supporting it via canonical's libappindicator. Unknown support for Unity8.

> Our plan is to remove that systray menu, or better, move it inside the toolbar in the remmina connection window.

I am using Linux Mint MATE which I understand is a fork of Gnome. I know this isn't an area to debate why a system tray feature is being depreciated, I'm all for making things better and standardised ... as long as the "new way" provides efficient ways to do the same thing.

> as written by @antenore , in the main remmina connection window, you can sort your hosts by "last use date/time", just click on its column. Frequently used hosts will appear at the top or at the bottom. This works only on the main remmina window, it does not work on the systray menu.

I find needing to have the main remmina window open unproductive. With the system tray, I know exactly where to move the mouse to access my list of connections, which are neatly grouped. If I had to navigate to the main remmina window, first I would need to determine if it was already open (system tray icon is always there) AND then I would need to locate the application on the taskbar, it will be in a different location depending on how many apps I have open (system tray icon is always in the same location).

**Comment by antenore** [comment]
OK, I'm thinking about it. I don't want more spaghetti code, so I prefer taking more time and do it the right way. Please be patient and if you come up with some ideas and/or code you are more than welcome.
